packages/paste-design-tokens/tokens/themes/dark/global/box-shadow.yml

```diff
@@ -18,13 +18,13 @@ props:
     value: "{!shadow-elevation-10}"
     comment: Shadow for cards.
   shadow-focus:
-    value: "{!offset-0} {!offset-0} {!offset-0} {!offset-20} {!palette-gray-0-transparent-50}"
+    value: "{!offset-0} {!offset-0} {!offset-0} {!offset-20} {!palette-gray-60}"
     comment: Shadow for focus ring on interactive elements.
   shadow-focus-inverse:
-    value: "{!offset-0} {!offset-0} {!offset-0} {!offset-20} {!palette-gray-0-transparent-40}"
+    value: "{!offset-0} {!offset-0} {!offset-0} {!offset-20} {!palette-gray-50}"
     comment: Shadow for focus ring on interactive elements on inverse backgrounds.
   shadow-focus-inset:
-    value: "inset {!offset-0} {!offset-0} {!offset-0} {!offset-10} {!palette-gray-0-transparent-50}"
+    value: "inset {!offset-0} {!offset-0} {!offset-0} {!offset-10} {!palette-gray-60}"
     comment: Shadow for inset focus on elements, such as DataGrid cells.
   shadow-border-weaker:
```

Comment:
Do shadow-border-weak and shadow-border-weaker need to match border colors? (gray-80 and gray-90)

cc @katieporter